app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/ProfileScreen.kt

```diff
@@ -88,7 +99,16 @@ fun ProfileScreen(navActions: NavigationActions, viewModel: ProfileViewModel) {
         // Toggle Button to switch between TOP SONGS and CHOSEN SONGS
         Button(
             onClick = { viewModel.changeChosenSongs() },
-            modifier = Modifier.testTag("toggleSongList")) {
+            modifier = Modifier.testTag("toggleSongList"),
+            colors =
+                ButtonColors(
+                    contentColor = Color.Black,
+                    containerColor =
+                        if (isTopSongsListVisible) MaterialTheme.colorScheme.primary
+                        else spotify_green,
+                    disabledContentColor = Color.Gray,
+                    disabledContainerColor = Color.Black),
+            shape = RoundedCornerShape(size = 10.dp)) {
```

Comment:
I agree but but it is somehow unclear how we can define one since I didn't wanna touch what was already there and as @mbouaissi mentioned the UI will come later when the Backend is done. So I am guessing this can be tolerated for now?

Reply:
Yeah I guess it's fine